requirements.txt

```diff
@@ -83,3 +83,69 @@ netaddr==0.8.0
 vulners==2.0.10
 fontawesomefree==6.4.0
 azure-devops==7.1.0b3
+
+
+#New Dependencies
```

Comment:
Please review version according to dependencies tree.

Reply:
Reviewed